src/components/Button/Button.test.js

```diff
@@ -56,5 +56,23 @@ describe('Button', () => {
         .toJSON()
       expect(tree).toMatchSnapshot()
     })
+    test('Array<ReactNode>', () => {
+      const tree = renderer
+        .create(
+          <Button.Medium.Black data-tid="foo" name="bar" disabled={false}>
+            {[
+              <div key="div1" />,
+              <div key="div2" />,
+              null,
+              undefined,
+              3,
+              'string',
+              <React.Fragment key="fragment">Fragment with key</React.Fragment>,
+            ]}
+          </Button.Medium.Black>
+        )
+        .toJSON()
+      expect(tree).toMatchSnapshot()
```

Comment:
oh yeah, this is exactly what I was talking about 👍 -- I'd say the markdown is a great local test to do and I'm glad you did, but maybe we should prune it from the PR

Reply:
will do